packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -44,11 +45,16 @@ const buildImg = () => (
   />
 );
 
-const MediaIndicatorComponent = (
-  <MediaIndicator duration="2:15" datetime="PT2M15S" service="news" />
+const MediaIndicatorComponent = (type, service) => (
+  <MediaIndicator
+    duration="2:15"
```

Comment:
We shouldn't render a duration when type isn't 'video' or 'audio'

<img width="80" alt="image" src="https://user-images.githubusercontent.com/10963046/61876366-965d8a80-aee4-11e9-98a8-f1c0c21ff6f7.png">